C/services/south/ingest.cpp

```diff
@@ -986,6 +986,21 @@ void Ingest::unDeprecateAssetTrackingRecord(AssetTrackingTuple* currentTuple,
 				m_logger->error("Failure while un-deprecating asset '%s'",
 						assetName.c_str());
 			}
+			else
+			{
+				string audit_details = "{\"asset\" : \"" + assetName +
+							"\", \"service\" : \"" + m_serviceName +
+							"\", \"event\" : \"" + event + "\"}";
+				// Add AuditLog entry
+				if (!m_mgtClient->addAuditEntry("ASTUN", "INFORMATION", audit_details))
+				{
+					m_logger->error("Failure while adding AuditLog entry " \
```

Comment:
Either keep it warning as we did for python while deprecating else make that error level as well